### DISCUSSION TITLE
Do you planning using MongoDB.EntityFrameworkCore in abp?

### DISCUSSION THREAD

**Discussion by P9avel** [abpframework/abp#17929] [General]
Hi. Now available https://www.nuget.org/packages/MongoDB.EntityFrameworkCore. Do you planning using MongoDB.EntityFrameworkCore in abp?

**Comment by maliming** [comment]
Thanks @P9avel 

I will study this library.

**Comment by maliming** [comment]
I checked. It has some limitations. I will keep my eyes open on it.

**Comment by P9avel** [comment]
Can you to describe in details?

**Reply by maliming** [reply]
https://github.com/mongodb/mongo-efcore-provider?tab=readme-ov-file#limitations--roadmap
